docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -94,6 +94,15 @@ The are additional fields available in the keycloak.status to facilitate keycloa
 
 In scenarios where advanced management is needed you may now directly update most fields on operator managed resources that have not been set by the operator directly. This can be used as an alternative to the unsupported stanza of the Keycloak spec. Like the unsupported stanza these direct modifications are not considered supported. If your modifications prevent the operator from being able to manage the resource, there Keycloak CR will show this error condition and the operator will log it.
 
+= Account Console v1
+
+The old Account Console (v1) is now completely removed. This version of the Account Console was marked as deprecated
```

Comment:
```suggestion
Account Console is now completely removed. This version of the Account Console was marked as deprecated
```

Reply:
I still think we need to refer to the v1 explicitly, we currently maintain 3 different versions, so users need to know what we're talking about.

Reply:
Okay,  then "The Account Console v1"
You don't need the parentheses if you don't provide them eslewhere.

Reply:
We used to refer to it as the "old account console", but more recently refer to it as "account console v1". Hence, why I used "old Account Console (v1)".

Reply:
Okay, I think that works too, @stianst